velox/dwio/common/SelectiveColumnReader.cpp

```diff
@@ -216,6 +216,9 @@ void SelectiveColumnReader::getIntValues(
           VELOX_FAIL("Unsupported value size: {}", valueSize_);
       }
       break;
+    case TypeKind::TIMESTAMP:
```

Comment:
Just override `TimestampColumnReader::getValues` and add it there instead

Reply:
Updated as below.

>   void getValues(RowSet rows, VectorPtr* result) override {
    getFlatValues<Timestamp, Timestamp>(rows, result, requestedType_);
  }